play/src/containers/GameInProgressContainer.tsx

```diff
@@ -37,6 +40,11 @@ export default function GameInProgressContainer({gameSession, teamId, currentSta
     setIsPregameCountdown(false);
   };
 
+  const handleUpdateScore = (inputScore: number) => {
```

Comment:
It should as the `updateTeamScore` method that it calls internally is async. I'm going to wait until the next PR that we discussed this morning to fix it, as it will be a lot cleaner when this logic is entirely contained within the `GameInProgressContainer`